test/mocks.go

```diff
@@ -1,44 +0,0 @@
-// Copyright 2015 ISRG.  All rights reserved
```

Comment:
Please do this change in another PR. Perhaps even one that is required for this PR to go out. This is a lot of churn.

I'm stopping my review here because of the amount of unrelated changes.